docusaurus/docs/ui-extensions/_category_.json

```diff
@@ -0,0 +1,6 @@
+{
+    "position": 8,
```

Comment:
I'd always expect to find reference (metadata) as the last topic in any table of contents, and troubleshooting as second last tbh. For me UI extensions should come before those. @josmperez any thoughts?

Reply:
Yes, "UI extensions" should come before Troubleshooting and Reference(s).